Assets/PlayroomKit/modules/Helpers/Helpers.cs

```diff
@@ -254,5 +257,36 @@ public static string SerializeObject(object value)
                     return default;
             }
         }
+
+        public static TurnData ParseTurnData(string json)
+        {
+            JSONNode jsonNode = JSON.Parse(json);
+            TurnData turnData = new TurnData
+            {
+                id = jsonNode["id"],
+                player = PlayroomKit.GetPlayerById(jsonNode["player"]["id"]),
+                data = jsonNode["data"]
+            };
+            return turnData;
+        }
+
+        public static List<TurnData> ParseAllTurnData(string json)
+        {
+            allTurns.Clear();
+            
+            JSONNode allData = JSON.Parse(json);
+            for (int i = 0; i < allData.Count; i++)
+            {
+                TurnData turnData = new TurnData
```

Comment:
Can't we call the function `ParseTurnData` here, which you've already defined above?